<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PRATIK" failures="1" tests="4" name="personal loan" time="0.079" errors="0" timestamp="2022-12-04T11:05:51 IST">
  <testcase classname="testNG.Ploan" name="pLoan" time="0.012"/>
  <testcase classname="testNG.day1" name="demo" time="0.001"/>
  <testcase classname="testNG.day1" name="failTest" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.day1.failTest(day1.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- failTest -->
  <testcase classname="testNG.day1" name="secondTest" time="0.002"/>
</testsuite> <!-- personal loan -->
